.github/dependabot.yml

```diff
@@ -0,0 +1,42 @@
+version: 2
+updates:
+  # master
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "pip"
+      directory: "/decrypt_oracle"
```

Comment:
Note, according to https://github.com/dependabot/feedback/issues/113, any .txt file at or one level below `directory` should be looked at.